x/concentrated-liquidity/tick_test.go

```diff
@@ -121,3 +121,206 @@ func (s *KeeperTestSuite) TestNextInitializedTick() {
 		})
 	})
 }
+
+func (s *KeeperTestSuite) TestInitOrUpdateTick() {
+	type param struct {
+		poolId      uint64
+		tickIndex   int64
+		liquidityIn sdk.Dec
+		upper       bool
+	}
+
+	tests := []struct {
+		name                   string
+		param                  param
+		tickExists             bool
+		expectedLiquidityNet   sdk.Dec
+		expectedLiquidityGross sdk.Dec
+		expectErr              bool
+	}{
+		{
+			name: "Init tick 50 with 50000000000 liquidity, upper",
+			param: param{
+				poolId:      1,
```

Comment:
nit: we can make it as `const validPoolId = 1` and reuse

Reply:
Sorry I may be missing something, but we cant just remove this as a param and set it as a constant since we have a test case that tests an invalid poolId of 2 right? How would I overwrite the const in this case?

Reply:
Ohh I see what you are saying now, yeah I can change this